pkg/platform/aws/aws.go

```diff
@@ -287,6 +288,30 @@ func (c *config) checkValidConfig() hcl.Diagnostics {
 	return diagnostics
 }
 
+// checkDifferentPortForMuliplePools checks if user is using multiple worker
+// pools, then they should have lb_http_port and lb_https_port flag set
+// with different values.
+func (c *config) checkDifferentPortForMuliplePools() hcl.Diagnostics {
+	var diagnostics hcl.Diagnostics
+
+	portMap := make(map[int]bool)
+
+	for _, wp := range c.WorkerPools {
+		if !portMap[wp.LBHTTPPort] && !portMap[wp.LBHTTPSPort] {
```

Comment:
If 2nd worker pool has port 80 or 443 specified, this code will still let trough the configuration, which is wrong.